### PR TITLE
🏛 Migrate ENS GraphQL queries from Apollo to Rainbow GraphQL Client

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-4457

## What changed (plus any additional context for devs)

This PR migrates the ENS GraphQL queries from Apollo Client to our new GraphQL Client w/ codegen.

## Screen recordings / screenshots

Everything is still the same.

## What to test

Make sure ENS functionalities still work.

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]

File: src/handlers/ens.ts [hunk 1]
```diff
@@ -188,28 +167,32 @@ export const fetchEnsTokens = async ({
...
+          const token = buildEnsToken({
+            contractAddress,
+            imageUrl: `https://metadata.ens.domains/mainnet/${contractAddress}/${tokenId}/image`,
+            name: registration.domain.name || '',
+            tokenId,
+          });
+          return token;
+        })
+        .filter(
+          <TToken>(token: TToken | null | undefined): token is TToken => !!token
```
Comment (estebanmino) [inline src/handlers/ens.ts:194]: was the double T of TToken intended?

File: src/hooks/useAccountENSDomains.ts [hunk 2]
```diff
@@ -21,18 +21,25 @@ async function fetchAccountENSDomains({
...
-    return domain;
-  });
+  return domains
+    .map(domain => {
+      if (!domain) return;
+      domain.name && prefetchENSAvatar(domain.name, { cacheFirst: true });
+      return domain;
+    })
+    .filter(
+      <TDomain>(domain: TDomain | null | undefined): domain is TDomain =>
```
Comment (estebanmino) [inline src/hooks/useAccountENSDomains.ts:40]: ah yes, i see the additional T on TDomain as well

**Review by estebanmino — APPROVED** [review]
built the app and looking good 🚀

File: src/hooks/useAccountENSDomains.ts (hunk 2 above)
Comment (jxom) [reply src/hooks/useAccountENSDomains.ts:40]: yep! generic syntax haha

**Pull request merged**